content/application.md

```diff
@@ -40,8 +40,9 @@ As soon as a match occurs, the network manager will send mails to the mentee(s)
 
 ## How to apply
 
-1. Click on the application link to fill the form.
-2. Answer all questions (Tip: be yourself, we want to know you and what you want to achieve with this mentorship).
-3. Review your answers and click submit.
+1.  Check the (application template)[#].
```

Comment:
Can you update with the link?

Reply:
I can't do that now because the application template PR hasn't been merged. I will do that later.

Reply:
I am not sure to understand why you can not add the link. The files are in this PR. Once you created the Vue files you will be able to check the pages and add the links there